### PR TITLE
First attempt at a gui

### PR DESCRIPTION
**Pull request by Milkshak3s**
Looks to be a solid direction to go with the UI (although it's a bit annoying to write).

- Refactored directory names so imports work
- Added setup.py
- Displays base stats
- Updates base stats on edit
- Can edit tonnage from UI
- Can edit max fuel from UI
- Added setter for Spacecraft tonnage
- Added setter for Spacecraft max_fuel
- Added getter for Spacecraft armor rating

### CODE REVIEW

File: imperium/shipyard.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        self.spacecraft.set_fuel(new_fuel)
+
+
+if __name__ == '__main__':
+    import sys
+
+    app = QApplication(sys.argv)
+    window = Window()
+    window.show()
+    sys.exit(app.exec_())
```
Comment (qu-gg) [inline imperium/shipyard.py]: Newline here, por favor

File: imperium/models/spacecraft.py [hunk 2]
```diff
@@ -189,6 +207,17 @@ def add_bridge(self):
...
+        :returns: An int representing the total armor
+        """
+        total_rating = 0
 
+        for armour_item in self.armour:
+            protection = armour_item.get('protection')
+            if protection is not None:
+                total_rating = total_rating + protection
 
+        return total_rating
```
Comment (qu-gg) [inline imperium/models/spacecraft.py]: Newline here (I think)

File: imperium/shipyard.py (hunk 1 above)
Comment (Milkshak3s) [reply imperium/shipyard.py]: This has newline, will show red return arrow if not (I think I've not realized this in the past as well)

Can verify by clicking the edit button for the file.

File: imperium/models/spacecraft.py (hunk 2 above)
Comment (Milkshak3s) [reply imperium/models/spacecraft.py]: Same thing here

**Pull request merged**